src/store/ConfigStore.ts

```diff
@@ -17,6 +17,9 @@ interface Config {
 
   symmetry: boolean;
   setSymmetry: (symmetry: boolean) => void;
+
+  tailwindcss: boolean;
+  setTailwindcss: (tailwindcss: boolean) => void;
```

Comment:
I think I would be tempted to name these slightly differently to make it obvious that it's a boolean to match it up with the checkbox label:

```suggestion
  useTailwindSyntax: boolean;
  setUseTailwindSyntax: (useTailwindSyntax: boolean) => void;
```